code/game/machinery/bots/secbot.dm

```diff
@@ -724,6 +724,9 @@ Auto Patrol: []"},
 /obj/machinery/bot/secbot/proc/assess_perp(mob/living/carbon/human/perp as mob)
 	var/threatcount = 0 //If threat >= 4 at the end, they get arrested
 
+	if(perp.alpha <= 0) //perp is invisible, hence innocent
```

Comment:
Use alpha <= 1 instead, there's one weird source of invisibility that uses alpha == 1.

Reply:
It would also be wise to add `if(see_invisible < perp.invisibility return 0)` one line under this, for future compatibility.

Reply:
figured there is some weird stuff going on with alpha somewhere, gonna add it now